recipes/gprofiler/meta.yaml

```
@@ -0,0 +1,57 @@
+{% set name = "gprofiler" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}-pkg
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/g/gprofiler-official/gprofiler-official-{{ version }}.tar.gz
+  sha256: 5015b47f10fbdcb59c57e342e815c9c07afbe57cd3984154f75b845ddef2445d
+
+build:
+  number: 0
+
+outputs:
+  - name: {{ name }}
+    script: build_base.sh
```

```suggestion
    script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
```
You don't need a separate build script in a recipe that uses such a simple build process.